src/app/components/Article.test.jsx

```diff
@@ -2,14 +2,20 @@ import React from 'react';
 import { shallow } from 'enzyme';
 import Article from './Article';
 
-describe('Article', () => {
-  it('renders the headline in an h1', () => {
-    const component = shallow(<Article />);
-    expect(component.find('h1').text()).toEqual('Article Headline');
-  });
+const HEADLINE = 'Article Headline';
+
+const expectElementTextToEqual = (element, value) => {
+  const component = shallow(<Article />);
+  expect(component.find(element).text()).toEqual(value);
+};
 
-  it('renders the title', () => {
-    const component = shallow(<Article />);
-    expect(component.find('title').text()).toEqual('Article Headline');
+const testElementTextValue = (testTitle, element, value) => {
+  it(testTitle, () => {
+    expectElementTextToEqual(element, value);
   });
+};
+
+describe('Article', () => {
+  testElementTextValue('renders the headline in an h1', 'h1', HEADLINE);
+  testElementTextValue('renders the title', 'h1', HEADLINE);
```

Comment:
testElementTextValue('renders the title', 'h1', HEADLINE);
shouldn't this be
testElementTextValue('renders the title', 'title', HEADLINE);
?